src/qtism/runtime/storage/binary/AbstractQtiBinaryStorage.php

```diff
@@ -53,6 +53,8 @@
  */
 abstract class AbstractQtiBinaryStorage extends AbstractStorage
 {
+    const VERSION = 1;
```

Comment:
Shouldn't we apply https://semver.org/ here?

Reply:
I would say it's overkill for a binary protocol